wiki/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:18.04
```

Comment:
Would it be possible to bump to 20.04 (the one used in the project)?

Reply:
Unfortunately the dependency graph of the underlying base wiki platform (`gollum`) is pretty intricate, hence `ubuntu:18.04` is the stable choice currently.